tests/integration/conftest.py

```diff
@@ -201,6 +215,24 @@ def clean_up(self):
         self.remove_keys_and_certificates()
 
 
+@pytest.fixture()
+def fixture_subman():
+    """
+    Fixture.
+    Set up the subscription manager on the system. Wrapper around SubscriptionManager class and its methods.
+    By default sets the subscription manager to the stagecdn (needed for SCA Enabled accounts). If you want
```

Comment:
I'll try to test it for the els

Reply:
The ELS needs to have the stagcdn set in sub man config, otherwise we are getting the Unauthorized error. After offline discussion we have decided to keep it as it is, only changing the doc string